### DISCUSSION TITLE
https://ericswpark.com/blog/2023/2023-11-29-hosting-my-own-short-urls-with-yourls-and-docker/

### DISCUSSION THREAD

**Discussion by giscus** [ericswpark/ericswpark.github.io-comments#9] [Announcements]
# https://ericswpark.com/blog/2023/2023-11-29-hosting-my-own-short-urls-with-yourls-and-docker/

Student Developer

https://ericswpark.com/blog/2023/2023-11-29-hosting-my-own-short-urls-with-yourls-and-docker/

**Comment by bodryj** [comment]
Don't work. The containers are installed, it is impossible to launch the swag config and further install yourls.

**Reply by ericswpark** [reply]
Hm, how does it not work? Once you have SWAG set up, you do have to edit one of the proxy configuration files in `proxy-confs/`, but that should be outlined in the SWAG README. It's really simple to set up.
